core/services/log/helpers_test.go

```diff
@@ -207,7 +211,7 @@ type simpleLogListener struct {
 	t                   *testing.T
 	db                  *gorm.DB
 	jobID               int32
-	skipMarkingConsumed bool
+	skipMarkingConsumed atomic.Bool
```

Comment:
It's hilarious but apparently architectures exist that can't atomically set/load a bool without help

Reply:
This whole series is a good read: https://research.swtch.com/hwmm